### DISCUSSION TITLE
How to auto convert window path into linux path?

### DISCUSSION THREAD

**Discussion by lawweiliang** [Eugeny/tabby#8156] [Q&A]
Operating System: Window 10 Home edition
Tabby:  1.0.196

---

### Problem
Copy the window path and paste it into git bash, the path remain in window path.
![image](https://user-images.githubusercontent.com/34936929/228164409-26320b3c-00d9-4010-b491-79502929728b.png)

### Goal
In `cmder`, it's auto convert the `window path` into `linux path`.
![image](https://user-images.githubusercontent.com/34936929/228164784-ec6414ad-67c6-469f-9042-af7fb31e4864.png)

### Question
Is there any setting for this?

---

**Comment by opticyclic** [comment]
ConEmu has this feature as well. 
It is quite frustration having to manually type in a long path instead of being able to copy from the Windows Explorer address bar
